README.md

```diff
@@ -12,13 +11,13 @@ Yeoman based generator for Symphony Bots and Applications:
 ## Installation
 Install Yeoman command line utility
 ```shell
-npm install -g yo
+npm i -g yo
```

Comment:
I would even put the npm installs on a single command to avoid copy pasting twice
(personally I would keep keep the _install_ as it is more explicit)